app/src/main/java/org/simple/clinic/overdue/callresult/CallResult.kt

```diff
@@ -87,7 +95,9 @@ data class CallResult(
         removeReason = removeReason,
         outcome = Outcome.RemovedFromOverdueList,
         timestamps = Timestamps.create(clock),
-        syncStatus = syncStatus
+        syncStatus = syncStatus,
+        patientId = appointment.patientUuid,
+        facilityId = appointment.facilityUuid
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":79,"endLine":100,"prLink":"https://app.graphite.dev/github/pr/simpledotorg/simple-android/4218"}} -->
<i class='graphite__hidden'>[Re: lines 79 to 100]<br/></i>

Same comment as above<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4218?utm_source=unchanged-line-comment">Graphite</a>.</span>